pkg/internal/wordpress/pod_template.go

```diff
@@ -540,6 +580,8 @@ func (wp *Wordpress) WebPodTemplateSpec() (out corev1.PodTemplateSpec) {
 					},
 				},
 			},
+			ReadinessProbe: wp.readinessProbe(),
+			LivenessProbe:  wp.livenessProbe(),
 		},
```

Comment:
After rebasing those ended up in the wrong place. For more details see drone build that fails :smile: 
```suggestion
		},
		ReadinessProbe: wp.readinessProbe(),
		LivenessProbe:  wp.livenessProbe(),
```